OpenRA.Game/Settings.cs

```diff
@@ -241,6 +241,19 @@ public Hotkey GetSupportPowerHotkey(int index)
 		}
 	}
 
+	public class IrcSettings
+	{
+		public string Hostname = "chat.freenode.net";
```

Comment:
Freenode may not be happy with us using their network for off-topic videogame chat.